doc/en/components/grids/_shared/cell-editing.md

```diff
@@ -342,6 +342,43 @@ public webGridCellEditCellTemplate = (ctx: IgcCellTemplateContext) => {
     }
 
 ```
+
+<!-- ComponentStart: TreeGrid -->
+```ts
+
+constructor() {
+        var treeGrid = document.getElementById('treeGrid') as IgcTreeGridComponent;
+        var column1 = document.getElementById('column1') as IgcColumnComponent;
+        var column2 = document.getElementById('column2') as IgcColumnComponent;
+        var column3 = document.getElementById('column3') as IgcColumnComponent;
+
+        treeGrid.data = this.webGridCellEditSampleRoleplay;
+        column1.inlineEditorTemplate = this.webGridCellEditCellTemplate;
+        column2.inlineEditorTemplate = this.webGridCellEditCellTemplate;
+        column3.inlineEditorTemplate = this.webGridCellEditCellTemplate;
+}
+
```

Comment:
missing type definition of the webGridCellEditSampleRoleplay, e.g.
public webGridCellEditSampleRoleplay: any[];
public webGridCellEditSampleRoleplay: SomeDataModel[];